### ISSUE TITLE
Key template doesnt work correctly with class methods

### ISSUE THREAD

**Issue by Bohdan-Ilchyshyn** [Krukov/cashews#138]
I want to use a cache decorator with class methods and another class method to generate the key, but I need help to get it to work correctly.

Code example:

```python3
class T:
    
    @classmethod
    @cache.early(ttl="1h", early_ttl="45m", key="user:{cls.get_user_id(token)}")
    def cached_class_method(cls, token: str):
        # Do something with token
        return {"hello": "world"}
    
    @classmethod
    def get_user_id(cls, token: str):
        # Do something with token
        return 12
```

I tried with dot `key="user:{cls.get_user_id(token)}"` and with column `key="user:{cls:get_user_id(token)}"` but it doesn't work.

**Comment by Krukov** [comment]
Hi @Bohdan-Ilchyshyn 

Unfortunately cashews support only a simply lookup through dots , and can't call any methods ( only attributes ).
Cashews provide an ability to add your own functions to templates but it is also limited and do not cover your case.

But I think that is it possible to add feature like this, I will write an update after weekends.

**Comment by Krukov** [comment]
I released a new version ( 6.1.0 ) that have the solution for your case. That is not so  straight as you expected but I hope it will help: 
Now you can use raw objects in your custom template tags- that is allow you to register you tag and pass a raw parameter/class to it - here is full example how you can use it in your case:
```python 

import asyncio

from cashews import cache, default_formatter

cache.setup("redis://")


class T:

    @classmethod
    @cache.early(ttl="1h", early_ttl="45m", key="user:{cls:get_user_id({token})}")
    async def cached_class_method(cls, token: str):
        # Do something with token
        return {"hello": "world"}

    @classmethod
    def get_user_id(cls, token):
         # Do something with token
        return f"{cls.__name__}:user_id:{token}"


class D(T):
    pass


@default_formatter.register("get_user_id", preformat=False)
def get_user_id(cls, token):
    return cls.get_user_id(token)


print(asyncio.run(D.cached_class_method("token_value")))
# the key will be "early:v2:user:D:user_id:token_value" in this case 
``` 

I hope that example is clear but if not feel free to ask.
